starknet_devnet/blueprints/rpc/call.py

```diff
@@ -22,13 +22,22 @@ async def call(request: FunctionCall, block_id: BlockId) -> List[Felt]:
     if not state.starknet_wrapper.contracts.is_deployed(int(request["contract_address"], 16)):
         raise RpcError(code=20, message="Contract not found")
 
+    # exception handling using [rpc_felt(int(calldata_value, 1)) for calldata_value in request["calldata"]]
+    # does not work properly, so we use the following workaround
+    for calldata_value in request["calldata"]:
+        try:
+            rpc_felt(int(calldata_value, 16))
+        except (ValueError, TypeError):
+            raise RpcError(code=22, message="Invalid calldata") from None
```

Comment:
Instead of raising from `None`, you should do `except (ValueError, TypeError) as error`, and raise from `error`. Is `except`ing both `ValueError` and `TypeError` necessary?

Reply:
great idea! will try it

yeah i think this is necessary since `TypeError: int() can't convert non-string with explicit base` might occur for 
`["wtf"]`